accelerator/config.c

```diff
@@ -24,6 +24,22 @@ int get_conf_key(char const* const key) {
   return 0;
 }
 
+static void remove_nonvalue_dash(int* argc, char** argv) {
+  int const end = *argc;
+  for (int i = 1; i < end; i++) {
+    if ((strlen(argv[i]) == 2) && (!strncmp(argv[i], "--", 2))) {
+      *argc = end - 1;
+      if (end == (i + 1)) {
+        break;
+      }
+
+      for (int j = i; j < end - 1; j++) {
```

Comment:
Can you make use of [memmove](http://man7.org/linux/man-pages/man3/memmove.3.html), which allows overlapping?